### PR TITLE
feat(repo-cloner): Migrate from science-toolkit repo

### PR DESCRIPTION
**Pull request by alpiquero**

### CODE REVIEW

File: repo-cloner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+LABEL maintainer="Builder"
+
+# Build the binary statically.
+ENV CGO_ENABLED=0
+
+WORKDIR /app
+COPY go.* ./
+RUN go mod download
+COPY . .
+RUN go build -o user-repo-cloner .
```
Comment (guumaster) [inline repo-cloner/Dockerfile:12]: small fix: change binary name to `repo-cloner`

File: .github/workflows/build-release.yml [hunk 2]
```diff
@@ -335,6 +335,39 @@ jobs:
...
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Get Version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
+
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v2
+        with:
+          context: ./jupyter-gpu
```
Comment (guumaster) [inline .github/workflows/build-release.yml:364]: context is `./repo-cloner`

File: repo-cloner/Dockerfile (hunk 1 above)
Comment (alpiquero) [reply repo-cloner/Dockerfile]: Fixed!

File: .github/workflows/build-release.yml (hunk 2 above)
Comment (alpiquero) [reply .github/workflows/build-release.yml]: Fixed!

**Review by aacecandev — APPROVED** [review]

**Pull request merged**